Makefile

```diff
@@ -85,7 +85,7 @@ generate: controller-gen conversion-gen
 
 build: manifests
 	# Build on host so we can make use of the cache
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -ldflags "${LDFLAGS}" -a -o manager main.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -ldflags "${LDFLAGS}" -o manager main.go
```

Comment:
At first this gave me pause, but then I realized we were trying to leverage the build cache in GitHub Actions (though I clearly missed the cache when I switched over to macOS builds). We also don't set this in GoReleaser for the CLI. I think I am ok with it.